### ISSUE TITLE
Group drop down filter variables after user and system object

### ISSUE THREAD

**Issue by ggzengel** [Icinga/icingaweb2#3128]
The most time I change filters I don't need user variables.
In the drop down list there is no difference between system object and user values.
I don't know what happens if I declare <vars.status>? Does I have than 2x "Service Status" in the list?
Which one choosing?

You have already an underscore for separating system object from user vars.
I would change this
```
  <option value="_service_load_wload5">Service Load Wload5</option>
```
to
```
  <option value="_service_load_wload5">User Service: Load Wload5</option>
```

Part of my incingweb2:

```
  <option value="_service_load_wload5">Service Load Wload5</option>
  <option value="service_long_output">Service Long Output</option>
  <option value="_service_mailq_critical">Service Mailq Critical</option>
  <option value="_service_mailq_warning">Service Mailq Warning</option>
  <option value="service_max_check_attempts">Service Max Check Attempts</option>
  <option value="_service_mins">Service Mins</option>
  <option value="service_modified_service_attributes">Service Modified Service Attributes</option>
```

**Comment by lippserd** [comment]
Hi,

Thanks for the report. In one of our next releases we'll rework our filter editor where this issue will be part of. I don't have an exact schedule though.

Cheers,
Eric

**Comment by lippserd** [comment]
Hi,

This issue will be fixed as part of #3631.

Best,
Eric
